socrata/socrata_upload.py

```diff
@@ -1,11 +1,12 @@
-import os
-import requests
-import time
-from dbt.cli.main import dbtRunner
 import contextlib
 import io
 import json
+import os
+import time
+
 import pandas as pd
+import requests
+from dbt.cli.main import dbtRunner
 from pyathena import connect
 from pyathena.pandas.cursor import PandasCursor
 
```

Comment:
This diff is unrelated to the PR, and is simply a workaround to resolve the linting issue that is also fixed by https://github.com/ccao-data/data-architecture/pull/587.